### PR TITLE
bitbox: add num_rings option

### PR DESCRIPTION
**Pull request by gabriele-0201**

### CODE REVIEW

**Comment by gabriele-0201** [comment]
* **#255** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/255?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gabriele-0201 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: benchtop/src/cli.rs [hunk 1]
```diff
@@ -114,6 +114,13 @@ pub struct WorkloadParams {
     #[arg(long = "fetch-concurrency", short)]
     #[clap(default_value = "1")]
     pub fetch_concurrency: usize,
+
+    /// Number of io_uring instances
```
Comment (pepyakin) [inline benchtop/src/cli.rs:118]: Only used with Nomt backend?

File: nomt/src/options.rs [hunk 2]
```diff
@@ -47,4 +50,12 @@ impl Options {
     pub fn metrics(&mut self, metrics: bool) {
         self.metrics = metrics;
     }
+
+    /// Set the number of io_uring instances.
+    ///
+    /// Must be more then 1
```
Comment (pepyakin) [inline nomt/src/options.rs:56]: than.

Comment (pepyakin) [reply nomt/src/options.rs:56]: Why should it be more than one and not zero?

**Review by pepyakin — APPROVED** [review]

File: nomt/src/options.rs (hunk 2 above)
Comment (gabriele-0201) [reply nomt/src/options.rs:56]: because otherwise there will be input output at all

Comment (pepyakin) [reply nomt/src/options.rs]: sorry, I don't understand it. Can you rephrase?

Comment (gabriele-0201) [reply nomt/src/options.rs]: sorry, you're right, it must be more than *zero*

**Review by rphmeier — APPROVED** [review]
